pkgs/development/python-modules/pycryptodome/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, fetchurl, python, buildPythonPackage, gmp }:
+
+buildPythonPackage rec {
```

Comment:
I can move this into the top level python packages file - I put it here to match pycrypto since pycryptodome is a fork of pycrypto.  Open to thoughts on this.